zentral/contrib/santa/templates/santa/pick_rule_binary.html

```diff
@@ -76,6 +78,10 @@ <h2>Pick Santa rule binary</h2>
     {% endfor %}
   </tbody>
 </table>
+{% else %}
+<div class="text-center py-5">
```

Comment:
Sorry, one more thing. This message is displayed even if there was no search yet. You could use `{% if form.has_changed %}` to only display the message if there is a search.